grammars/php.cson

```diff
@@ -178,12 +178,15 @@
   'comments':
     'patterns': [
       {
-        'begin': '/\\*\\*(?:#@\\+)?\\s*$'
-        'captures':
-          '0':
+        'begin': '(/\\*\\*)(((#@\\+)?\\s*$)|\\s+)'
```

Comment:
I'm not sure if we should keep the #@+ syntax. 

The site that was referenced in the comment no longer exists and I could only find one site that mentioned the syntax and referenced the same broken link that was in the comment